frontend/src/pages/detail/DetailPage.tsx

```diff
@@ -1,11 +1,15 @@
 import Carousel from "./components/Carousel/Carousel";
+import TitleSection from "./components/TitleSection/TitleSection";
+import OverviewSection from "./components/OverviewSection/OverviewSection";
 import PlatformSection from "./components/PlatformSection/PlatformSection";
 import TechStacksSection from "./components/TechStacksSection/TechStacksSection";
 
 function DetailPage() {
   return (
     <div>
```

Comment:
```suggestion
    <main>
```

`DetailPage` 최상위에 있는 태그에 들어가는 css 속성이 없다면 프래그먼트로 사용하거나 위처럼 `main` 태그를 사용해도 좋을 것 같아요~

내부는 `section` 태그로 시맨틱하게 작성해주신 것 같네요!👍👍👍

Reply:
넵 좋습니다~!